Sources/web3swift/Web3/Web3+Structures.swift

```diff
@@ -353,38 +153,27 @@ public struct EventLog: Decodable {
         let address = try container.decode(EthereumAddress.self, forKey: .address)
         self.address = address
 
-        guard let blockNumber = try decodeHexToBigUInt(container, key: .blockNumber) else {throw Web3Error.dataError}
-        self.blockNumber = blockNumber
+        self.blockNumber = try container.decodeHex(to: BigUInt.self, key: .blockNumber)
 
-        guard let blockHash = try decodeHexToData(container, key: .blockHash) else {throw Web3Error.dataError}
-        self.blockHash = blockHash
+        self.blockHash = try container.decodeHex(to: Data.self, key: .blockHash)
 
-        guard let transactionIndex = try decodeHexToBigUInt(container, key: .transactionIndex) else {throw Web3Error.dataError}
-        self.transactionIndex = transactionIndex
+        self.transactionIndex = try container.decodeHex(to: BigUInt.self, key: .transactionIndex)
 
-        guard let transactionHash = try decodeHexToData(container, key: .transactionHash) else {throw Web3Error.dataError}
-        self.transactionHash = transactionHash
+        self.transactionHash = try container.decodeHex(to: Data.self, key: .transactionHash)
 
-        guard let data = try decodeHexToData(container, key: .data) else {throw Web3Error.dataError}
-        self.data = data
+        self.data = try container.decodeHex(to: Data.self, key: .data)
 
-        guard let logIndex = try decodeHexToBigUInt(container, key: .logIndex) else {throw Web3Error.dataError}
-        self.logIndex = logIndex
+        self.logIndex = try container.decodeHex(to: BigUInt.self, key: .logIndex)
 
-        let removed = try decodeHexToBigUInt(container, key: .removed, allowOptional: true)
-        if removed == 1 {
-            self.removed = true
-        } else {
-            self.removed = false
-        }
+        let removed = try? container.decodeHex(to: BigUInt.self, key: .removed)
+        self.removed = removed == 1 ? true : false
```

Comment:
Beyond removing the EthereumTransaction and TransactionOptions extensions from this file, all other changes were cosmetic lint changes, no code analysis was done. This was a file that was missed in the initial lint cleanup PR (or was subsequently reverted in a PR since), so the cleanup was included here again out of convenience. 

And while yes it could be shortened to what you show, it also potentially confuses the intent, and could be construed as an error by someone down the road. Sometimes it is best to leave the more explicit code for clarity. Having said that, I am not a fan of the ternary operator here either, and would like to see it restructured. However that will have to be for another PR, the intent of this PR is 1559 support.